### PR TITLE
moved `isArray` to util

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by bekzod** [comment]
I was going to do that but then didn't want to rush and wanted to get your approval :), doing that now

File: lib/backburner/utils.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ export function isNumber(suspect: any): suspect is number {
   return typeof suspect === 'number';
 }
 
+export const isArray = Array.isArray || function(suspect: any): suspect is T[] {
```
Comment (rwjblue) [inline lib/backburner/utils.ts]: Hmm.  I think we only support platforms with a native `Array.isArray`, no?

Comment (stefanpenner) [inline lib/backburner/utils.ts]: Ya it looks like its IE 9+, good catch.

**Comment by bekzod** [comment]
@stefanpenner should I close this and open new PR for using `Array.isArray`

**Comment by stefanpenner** [comment]
@bekzod feel free to also just update this one to use `Array.isArray` we can reuse this PR :)

**Pull request merged**